src/sources/mongodb_metrics/types.rs

```diff
@@ -323,10 +323,11 @@ pub struct CommandServerStatusWiredTigerBlockManager {
 pub struct CommandServerStatusWiredTigerCache {
     #[serde(rename = "bytes currently in the cache")]
     pub bytes_total: i64,
-    #[serde(rename = "bytes read into cache")]
-    pub bytes_read_into: i64,
-    #[serde(rename = "bytes written from cache")]
-    pub bytes_written_from: i64,
+    // Unused
```

Comment:
Can we just delete these?

Reply:
Probably? I wasn't certain the format wouldn't have problems with missing fields to deserialize. Otherwise, yes, I think they can go.

Reply:
It looks like it doesn't have `disallow_unknown_fields` so we should be ok. Tests should catch it.